proposals/010-will-fund-for-food.md

```diff
@@ -0,0 +1,89 @@
+# Will Fund For Food Proposal
+## Liquidity Advertisements
+
+## Problem Description
+
+Sourcing liquidity on the lightning network is a common problem
+for any node looking to accept inbound payments.
+
+## Overview
+
+This proposal adds a TLV to the `node_announcement`,
+which permits a node to advertise a rate for which they will contribute
+funds to an open channel request. Due to the nature of the protocol, these
+opens are only possible on the v2 channel protocol.
+
+Any advertised liquidity lease is for a duration of 4032 blocks, or
+approximately 28 days.
```

Comment:
Seems like it'd be better to keep this configurable. Is there any reason not to?

Reply:
Because the more variables you have the harder it is to computationally compare rates.  It's actually pretty easy to add a parameter for this later, if it proves popular, but I think we need to see what it looks like in practice (should the fee guaranty be separately configurable, for example).